### PR TITLE
docs: update "ca-" to "co-" files to match style guide

### PR DESCRIPTION
**Pull request by SarahLKatz**
## Description

For markdown files in the folder `/docs/docs` starting with ca- through co-, updates titles to use Title Case and changes "we/our" to "you/yours."

## Related Issues

Addresses Issue #18284

### CODE REVIEW

File: docs/docs/client-data-fetching.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ Compiling pages at [build-time](/docs/glossary#build) is useful when your websit
 
 Because a Gatsby site [hydrates](/docs/glossary#hydration) into a React app after loading statically, Gatsby is not just for static sites. You can also fetch data dynamically on the client-side as needed, like you would with any other React app.
 
-To illustrate this, we'll walk through a small example site that uses both Gatsby's data layer at build-time and data on the client at run-time. This example is based loosely on Jason Lengstorf's [Gatsby with Apollo](https://github.com/jlengstorf/gatsby-with-apollo) example. We'll be fetching character data for Rick (of Rick and Morty) and a random pupper image.
+To illustrate this, you'll walk through a small example site that uses both Gatsby's data layer at build-time and data on the client at run-time. This example is based loosely on Jason Lengstorf's [Gatsby with Apollo](https://github.com/jlengstorf/gatsby-with-apollo) example. You'll be fetching character data for Rick (of Rick and Morty) and a random pupper image.
```
Comment (marcysutton) [inline docs/docs/client-data-fetching.md]:
```suggestion
To illustrate this, check out a small example site that uses both Gatsby's data layer at build-time and data on the client at run-time. This example is based loosely on Jason Lengstorf's [Gatsby with Apollo](https://github.com/jlengstorf/gatsby-with-apollo) example, and will fetch character data for Rick (of Rick and Morty) and a random pupper image.
```

File: docs/docs/content-and-data.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Sourcing Content and Data
+title: Sourcing Content & Data
```
Comment (marcysutton) [inline docs/docs/content-and-data.md]:
```suggestion
title: Sourcing Content and Data
```
"and" is spelled out in the URL, so I'd keep this one as-is.

Comment (SarahLKatz) [reply docs/docs/content-and-data.md]: Thanks for the comments - I can make the changes and submit a new PR to better adhere to the preferred style.

**Comment by SarahLKatz** [comment]
@marcysutton - I made the changes related to your review and submitted them as PR #18473: 
https://github.com/gatsbyjs/gatsby/pull/18473

Thank you for the comments and insight!

**Pull request merged**